LoRaEngine/LoraKeysManagerFacade/LoRaADRFunction/LoRaADRRedisStore.cs

```diff
@@ -131,9 +133,9 @@ private async Task<LoRaADRTable> GetADRTableCore(string key)
             return null;
         }
 
-        private static string GetEntryKey(string devEUI)
+        private static string GetEntryKey(DevEui devEUI)
         {
-            return devEUI + CacheToken;
+            return devEUI.ToString("N", CultureInfo.InvariantCulture) + CacheToken;
```

Comment:
wouldn't it make sense to change the default to be "N" for the Eui?

Reply:
Tagging @atifaziz for his input here. It's related to some of your other comments, let's discuss it here.

Reply:
This was addressed with 6774e906970a25a27d7e163acf4f545376e454d9